resources/definitions/fdmprinter.def.json

```diff
@@ -3379,6 +3379,30 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "support_skip_some_zags":
+                {
+                    "label": "Skip Some ZigZags Connections",
```

Comment:
1. It's not zalled 'ZigZags', but 'Zig Zag'
2. In the near future we're going to make any setting zig-zaggable; I already made the setting as part of my cross infill. It's called 'Connect Infill Lines', so it would be logical if a similar setting for support would be called 'Connect Support Infill Lines'.
3. This setting doesn't mention it's about support

Solution:
rename to 'Skip Some Support Infill Line Connections'